index.html

```diff
@@ -602,35 +607,22 @@ <h1>Trust Model</h1>
 risk assessment of the verifier.
         </li>
         <li>
-All entities trust the <a>identifier registry</a> to be incorruptible and
-to be a correct record of which identifiers belong to which <a>entities</a>.
+All entities trust the <a>Verifiable data repository</a> to be un-corruptible and
+to be a correct record of which data is owned (and can be updated) by which
+<a>entities</a>.
         </li>
         <li>
-The <a>subject</a> trusts the <a>issuer</a> to issue true
+The <a>subject</a> trusts (since only the holder can verify) the <a>issuer</a> to issue true (i.e., not false)
```

Comment:
We wanted to emphasize that the subject cannot verify the credential, no one apart from holder can. Subject can only verify if he is the holder

Reply:
I reject this proposed change, because verification is nothing to do with this trust statement.
This trust statement is stating a fact, namely, that the subject trusts the issue to issue true statements about him or her. Period.
Verification is not within the scope of this sentence and is irrelevant.